ui-react/packages/atlasmap-provider/src/components/toolbar/toolbar-util.ts

```diff
@@ -22,29 +22,44 @@ export async function deleteAtlasFile(fileName: string, isSource: boolean) {
   cfg.initializationService.updateInitComplete();
 }
 
+/**
+ * Correct the AtlasMap mappings catalog file name by giving it the correct
+ * suffix if needed.
+ *
+ * @param fileName - user-specified catalog file name
+ */
+ function handleMappingsInstanceName(fileName: string): string | undefined {
+
+  if (fileName !== null && fileName.length > 0) {
```

Comment:
File name can’t be null, so neither the return of this function can. This should simplify the code somewhere - I saw an ! that can be removed